### DISCUSSION TITLE
Build up Rust environments

### DISCUSSION THREAD

**Discussion by carloscabanero** [blinksh/blink#1726] [Show and tell]
A place to discuss how to [Build up Rust environments](https://docs.blink.sh/build-up/rust).

**Comment by jasonjurotich** [comment]
@carloscabanero I don't know why I am just seeing this now... I think it would be awesome if you could make a video about what you can do with rust on an iPad for YouTube. I really don't think enough people know about this. I would really like to know the possibilities and limitations, etc. 

I use Helix daily for programming, and most of my work in Rust is backend, using, for example SurrealDB, reqwest, etc. Would it be possible to program webservers from an iPad locally with this, debugging locally in Helix, and using claude code, or gemini cli, or goose? 

I really think this should be promoted more if it were actually possible... Any comments would be appreciated...
